### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by Yuriy27**
Now server is able to process requests

### CODE REVIEW

File: softgroup/frontend/src/main/java/com/softgroup/rest/config/RestApplicationConfiguration.java [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+
+import org.springframework.context.annotation.ComponentScan;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.context.annotation.FilterType;
+
+/**
+ * Created by yuriy on 08.03.17.
+ */
+@Configuration
+@ComponentScan(basePackages = "com.softgroup",
```
Comment (Dominys) [inline softgroup/frontend/src/main/java/com/softgroup/rest/config/RestApplicationConfiguration.java]: @Yuriy27 This configuration should start only rest module.

File: softgroup/services/authorization/authorization-imp/src/main/java/com/softgroup/authorization/impl/AuthorizationRouter.java [hunk 2]
```diff
@@ -1,18 +1,19 @@
...
 import org.springframework.stereotype.Component;
 
 /**
  * Created by yuriy on 28.02.17.
  */
 @Component
 public class AuthorizationRouter
-        extends AbstractRouterHandler<RequestHandlerFactory<AuthorizationRequestHandler>>
-        implements AuthorizationRouterHandler {
+        extends AbstractRouterHandler<AuthorizationHandlerFactory>
```
Comment (Dominys) [inline softgroup/services/authorization/authorization-imp/src/main/java/com/softgroup/authorization/impl/AuthorizationRouter.java:12]: @Yuriy27 We should use interface (factory), not implementation

**Comment by Dominys** [comment]
@Yuriy27 Did i approve this PR ???

**Comment by Yuriy27** [comment]
I did changes that you've said me to do and i have marged it. Now I use interface of fabric(not implementation) and raise only rest module. This code was tested with postman and it works.

**Comment by Dominys** [comment]
@Yuriy27 But i did not approve this!

**Pull request merged**